apps/shelter/src/components/layout/index.tsx

```diff
@@ -13,7 +13,13 @@ export default function Layout() {
   };
 
   return (
-    <Container maxW="container.sm" h="100vh" p={0} centerContent>
+    <Container
+      position="relative"
```

Comment:
```suggestion
      pos="relative"
```
이런 식으로 줄임말을 활용하면 좋을 것 같아요!!

Reply:
줄임말이 있다는 것을 깜빡하고 있었네요~ 수정하도록 하겠습니다!